doc/classes/Image.xml

```diff
@@ -322,6 +345,7 @@
 			<argument index="2" name="interpolation" type="int" enum="Image.Interpolation" default="1">
 			</argument>
 			<description>
+				Resizes the image to a new size specified with [code]width[/code] and [code]height[/code]. The image is stretched to the new size and the data is interpolated according to the interpolation method specified with [code]interpolation[/code].
```

Comment:
> Resizes the image to a new size specified with [code]width[/code] and [code]height[/code].

Resizes the image to the given [code]width[/code] and [code]height[/code].

- "stretched" implies it only sizes upward.

> The image is stretched to the new size and the data is interpolated according to the interpolation method specified with [code]interpolation[/code].

New pixels are calculated using [code]interpolation[/code]. See [code]INTERPOLATE_*[/code] constants.